contracts/Gap.sol

```diff
@@ -201,7 +201,7 @@ contract Gap is Initializable, OwnableUpgradeable, EIP712Upgradeable {
                 .multiRequest;
             // If first item reference an attestation, checks if sender
             // is owner or attester of that attestation.
-            validateCanAttestToRefs(request.data, attester);
+            // validateCanAttestToRefs(request.data, attester); // Override to skip checks
```

Comment:
General comment everywhere in the codebase, remove lines instead of commenting, we have history anyway, can lookup the code if needed.